src/lib/app/app-icon/app-icon.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { IApp } from '..';
+
+@Component({
+  selector: 'app-app-icon',
```

Comment:
Can you rename this selector to `vui-app-icon` please?

Reply:
or maybe `vtd-`? haha

Reply:
`vui-app-icon` lol